### PR TITLE
beanshell 2.0b4 (new formula)

### PR DESCRIPTION
**Pull request by kadwanev**
- [X ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/beanshell.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Beanshell < Formula
+  desc "Java Beanshell - Lightweight Scripting for Java"
+  homepage "http://www.beanshell.org/"
+  url "http://www.beanshell.org/bsh-2.0b4.jar"
```
Comment (Moisan) [inline Formula/beanshell.rb]: The latest version is `bsh-2.0b6`. Also, according to the homepage, upstream releases are now on Github so we should use the releases from https://github.com/beanshell/beanshell/releases.

File: Formula/beanshell.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  sha256 "91395c07885839a8c6986d5b7c577cd9bacf01bf129c89141f35e8ea858427b6"
+
+  bottle :unneeded
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install "bsh-#{version}.jar"
+
+    (bin/"bsh").write <<~EOS
```
Comment (Moisan) [inline Formula/beanshell.rb]: No need to create a bash script here, instead use 
```
 bin.write_jar_script libexec/"bsh-#{version}.jar", "bsh"
```

**Review by Moisan — CHANGES_REQUESTED** [review]

**Comment by zbeekman** [comment]
Ping @kadwanev

**Comment by fxcoudert** [comment]
Given no response, closing for now.

**Pull request closed without merging**